<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="chloe.local" name="com.pokitdok.tests.PokitDokTest" tests="10" failures="1" timestamp="21 Jul 2014 16:25:04 GMT" time="0.901" errors="0">
  <testcase name="providersTest" time="0.062" classname="com.pokitdok.tests.PokitDokTest">
    <failure type="java.lang.ClassCastException" message="org.json.simple.JSONArray cannot be cast to java.util.Map">
      <![CDATA[java.lang.ClassCastException: org.json.simple.JSONArray cannot be cast to java.util.Map
	at com.pokitdok.tests.PokitDokTest.assertHasData(PokitDokTest.java:125)
	at com.pokitdok.tests.PokitDokTest.providersTest(PokitDokTest.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.TestNG.privateMain(TestNG.java:1364)
	at org.testng.TestNG.main(TestNG.java:1333)
]]>
    </failure>
  </testcase> <!-- providersTest -->
  <testcase name="eligibilityTest" time="0.461" classname="com.pokitdok.tests.PokitDokTest"/>
  <testcase name="cashPricesTest" time="0.032" classname="com.pokitdok.tests.PokitDokTest"/>
  <testcase name="activitiesTest" time="0.198" classname="com.pokitdok.tests.PokitDokTest"/>
  <testcase name="claimsTest" time="0.029" classname="com.pokitdok.tests.PokitDokTest"/>
  <testcase name="insurancePricesTest" time="0.040" classname="com.pokitdok.tests.PokitDokTest"/>
  <testcase name="shouldCreateTest" time="0.000" classname="com.pokitdok.tests.PokitDokTest"/>
  <testcase name="payersTest" time="0.058" classname="com.pokitdok.tests.PokitDokTest"/>
  <testcase name="claimsStatusTest" time="0.021" classname="com.pokitdok.tests.PokitDokTest"/>
  <testcase name="filesTest" time="0.000" classname="com.pokitdok.tests.PokitDokTest"/>
</testsuite> <!-- com.pokitdok.tests.PokitDokTest -->
